react/features/base/sounds/actions.js

```diff
@@ -84,17 +102,21 @@ export function playSound(soundId: string): Object {
  * created for given source object.
  * @param {string} soundName - The name of bundled audio file that will be
  * associated with the given {@code soundId}.
+ * @param {Object} options - Optional paramaters.
  * @returns {{
  *     type: REGISTER_SOUND,
  *     soundId: string,
- *     src: string
+ *     src: string,
+ *     options: Object
  * }}
  */
-export function registerSound(soundId: string, soundName: string): Object {
+export function registerSound(
+        soundId: string, soundName: string, options: Object = {}): Object {
```

Comment:
I really like that you added a config object instead of a simple boolean! Not a must have, just wondering: wouldn't it be good to have a list of supported options here instead of an empty object? A bit like there is one in the base/profile or base/settings actions.js